web/config/sync/search.page.node_search.yml

```diff
@@ -0,0 +1,15 @@
+uuid: 82431853-112f-46c2-a5bd-fd4f696ebc23
+langcode: en
+status: true
```

Comment:
Should the node search page be enabled? It is not used and we do not enable user search either.